importer/v8/importer.go

```diff
@@ -153,18 +157,36 @@ func (i *Importer) processDDL(scanner *bufio.Scanner) {
 		if strings.TrimSpace(line) == "" {
 			continue
 		}
-		i.queryExecutor(line)
+
+		i.createDatabaseQuery = line
 	}
 }
 
-func (i *Importer) processDML(scanner *bufio.Scanner) {
+// processDML actually processes each of the points once it has created the target database. The database can
+// either be specified by the user, or it will be read from the import file.  The same goes for the retention
+// policy it too can be overriden by specifying `rp` or else it too is set to whatever is spcfied in the DML.
+func (i *Importer) processDML(scanner *bufio.Scanner, dboverride string, rpoverride string) {
+
+	// If a user specified a dboverride, override the command specified in the DDL
+	if dboverride != "" {
+		i.createDatabaseQuery = fmt.Sprintf("CREATE DATABASE %s", dboverride)
+		i.database = dboverride
+	}
+	i.queryExecutor(i.createDatabaseQuery)
```

Comment:
I'm unclear on how that happens because the processDDL just finds the create database statement and sets it for processing later. It used to be the case that processDDL creates the database but in the PR that is no longer the case. From the previous snippet:

` i.createDatabaseQuery = line `